src/main/scala/com/atomist/rug/kind/python3/PythonFileType.scala

```diff
@@ -36,34 +36,39 @@ object PythonFileType {
 import com.atomist.rug.kind.python3.PythonFileType._
 
 class PythonFileType(
-                  evaluator: Evaluator
-                )
+                      evaluator: Evaluator
+                    )
   extends Type(evaluator)
-    with ReflectivelyTypedType {
+    with ReflectivelyTypedType
+    with ContextlessViewFinder {
 
   def this() = this(DefaultEvaluator)
 
   override def description = "Python file"
 
+  override def resolvesFromNodeTypes = Set("File")
+
   override def viewManifest: Manifest[MutableContainerMutableView] = manifest[MutableContainerMutableView]
 
   override protected def findAllIn(rugAs: ArtifactSource,
                                    selected: Selected,
                                    context: TreeNode,
                                    poa: ProjectOperationArguments,
-                                   identifierMap: Map[String, Object]): Option[Seq[MutableView[_]]] = {
+                                   identifierMap: Map[String, Object]): Option[Seq[TreeNode]] = {
     context match {
       case pmv: ProjectMutableView =>
         Some(pmv.currentBackingObject
```

Comment:
Why is `resolvesFromNodeTypes` only "File" but `findAllIn` can find from both Project and File MutableViews?